.travis.yml

```diff
@@ -2,5 +2,7 @@ language: node_js
 node_js:
   - 0.8
   - 0.10
+  - 0.12
+  - iojs
 before_install:
   - npm install -g npm@~1.4.6
```

Comment:
Only an addition is shown here—the dashes are part of the YAML syntax itself. :)